app/handlers/personal/start.py

```diff
@@ -159,6 +159,14 @@ async def add_words_to_study(callback_query: types.CallbackQuery, callback_data:
         await callback_query.answer('Added to study.')
 
 
+async def my_variant(callback_query: types.CallbackQuery):
```

Comment:
Function is missing a return type annotation  [no-untyped-def] mypy(error)

Reply:
add -> None